Osiris/Hacks/Misc.cpp

```diff
@@ -506,3 +506,64 @@ void Misc::killMessage(GameEvent& event) noexcept
     cmd += "\"";
     interfaces.engine->clientCmdUnrestricted(cmd.c_str());
 }
+
+void Misc::drawBombDamage() noexcept
+{
+    const auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
+
+    //No Alive return since it is useful if you want to call it out to a mate that he will die
+    if (!localPlayer || !config.misc.bombDamage)
+        return;
+
+    for (int i = interfaces.engine->getMaxClients(); i <= interfaces.entityList->getHighestEntityIndex(); i++)
+    {
+        auto entity = interfaces.entityList->getEntity(i);
+        if (!entity || entity->isDormant() || entity->getClientClass()->classId != ClassId::PlantedC4 || !entity->c4Ticking())
+            continue;
+
+        auto vecBombDistance = entity->origin() - localPlayer->origin();
+
+        const auto a = 450.7f;
```

Comment:
magic numbers.
excessive use of auto.
a, b, and c are only used once so why do they need to be a variable?

Reply:
No idea ask the original author about these.

Reply:
no need, i was just lazy as my name may indicate